packages/hub/src/app/new/group/NewGroup.tsx

```diff
@@ -60,7 +60,7 @@ export const NewGroup: FC = () => {
   });
 
   return (
-    <form onSubmit={onSubmit}>
+    <form onSubmit={() => onSubmit()}>
```

Comment:
why the change?

Reply:
Last item in https://github.com/quantified-uncertainty/squiggle/pull/2498#discussion_r1385665972 comment